src/main/resources/liquibase/DDP-5672-longitudinal-wait.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet author="andrew" id="DDP-5672-longitudinal-wait">
+        <addColumn tableName="ddp_kit_request_settings">
+            <!-- if set, longitudinal kits should not be sent out unless a previous kit for the ptp has come back within at most this many days -->
+            <column name="longitudinal_max_wait_days" type="int"/>
+        </addColumn>
+        <addColumn tableName="ddp_kit_request">
+            <!-- time at which an order was transmitted to an external system -->
+            <column name="order_transmitted_at" type="datetime(6)"/>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="andrew" author="DDP-5672-backfill">
+        <!--
+            set approximate order date for tbos gbf orders.  if null, set a bogus date
+            so that new code does not accidentally re-order the kit.
+         -->
+        <sql>
+            update ddp_kit_request_settings set longitudinal_max_wait_days=30 where external_shipper='gbf'
+        </sql>
```

Comment:
usually because of safe update mode we need to update by the id..., I am not sure if that is the case by liquibase too

Reply:
I thought safe mode only kicks in when you have a blank `where` clause?  We'll find out...